snaps/learn/tutorials/gas-estimation.md

```diff
@@ -28,103 +28,146 @@ starter kit by running:
 yarn create @metamask/snap gas-estimation-snap
 ```
 
-or
-
-```bash
-npx @metamask/create-snap gas-estimation-snap
-```
-
-or
-
-```bash
-npm create @metamask/snap gas-estimation-snap
-```
-
 Next, `cd` into the `gas-estimation-snap` project directory and run:
 
 ```bash
 yarn install
 ```
 
-This initializes your development environment with the required dependencies.
-You may get a warning similar to the following:
+This initializes your development environment with the required dependencies. 
+
+<details>
+  <summary>Did you get a warning?</summary>
+  <div>
+You may get a warning such as: 
 
 ```bash
 @lavamoat/allow-scripts has detected dependencies without configuration. explicit configuration required.
 run "allow-scripts auto" to automatically populate the configuration.
 ```
-
-You can resolve this error by running the following command:
-
+You can resolve this error by running:
+    
 ```bash
 yarn run allow-scripts auto
 ```
+ </div>
+</details>
+
+
+### 2. (Optional) customize your Snap
+
+This Snap is generated from a boilerplate typescript example Snap. While the following steps in the 
+drop-down are optional for testing, they significantly improve the UX by aligning the Snap with its function.
+
+<details>
+  <summary>Customize the Snap's UX</summary>
+  <div>
+    2.1 Update your Snap to [display an icon](../best-practices/design-guidelines.md#optimize-your-metadata) in MetaMask.
```

Comment:
```suggestion
#### 2.1. Display an icon

Update your Snap to [display an icon](../best-practices/design-guidelines.md#optimize-your-metadata) in MetaMask.
```

Also, add `toc_max_heading_level: 4` to the page metadata to make these headings visible in the TOC.

Reply:
Hmm -- so I do prefer the headings for the steps, but worried that adding them to toc makes the tutorials look more imposing, was trying to give a more welcoming flow. For sure, the div has to go if we toc4 cause the links don't work into divs. Will do that now, come back to me on toc4 decision.